src/main/java/reposense/wizard/BasicWizard.java

```diff
@@ -0,0 +1,22 @@
+package reposense.wizard;
+
+import reposense.wizard.prompt.OptionalSincePrompt;
+import reposense.wizard.prompt.OptionalUntilPrompt;
+import reposense.wizard.prompt.OptionalViewPrompt;
+import reposense.wizard.prompt.Prompt;
+import reposense.wizard.prompt.RepoPrompt;
+
+/**
+ * Represents a basic wizard run.
+ */
+public class BasicWizard extends Wizard {
+    private static final Prompt[] INITIAL_PROMPTS = new Prompt[] {
+            new OptionalSincePrompt(),
+            new OptionalUntilPrompt(),
+            new OptionalViewPrompt(),
+            new RepoPrompt()
+    };
```

Comment:
I tried it myself and I don't think it makes a lot of sense to put the repo last. I think it should be first since I'm usually thinking of what I want to analyze and then the time interval to analyze rather than the other way around

Reply:
I agree, lets change it to be the first.